Add multi-instance RPMsg and IPC Service (#36120)
zephyrproject-rtos/zephyr: samples/subsys/ipc/rpmsg_multi_instance/README.rst

.. _Multiple_instance_RPMsg_sample:

Multiple instance of RPMsg sample Application

> ################################

Reviewer: Title underline too short.

Author: Fixed.

> Multiple instance of RPMsg sample Application
> #############################################

Reviewer: ```suggestion
Multiple instance of RPMsg
##########################
```

Overview
********

> Multiple instance of RPMsg is an abstraction created over OpenAMP that makes initialization
> and endpoints creation process easier.
> This application demonstrates how to use multi-instance RPMsg in Zephyr.

Reviewer: ```suggestion
Multiple instance of RPMsg is an abstraction created over OpenAMP.
It simplifies the  initialization and endpoint creation process.
This sample demonstrates how to use multi-instance RPMsg in Zephyr.
```

Building the application for nrf5340dk_nrf5340_cpuapp
*****************************************************

.. zephyr-app-commands::
   :zephyr-app: samples/subsys/ipc/rpmsg_multi_instance
   :board: nrf5340dk_nrf5340_cpuapp
   :goals: debug

> Open a serial terminal (minicom, putty, etc.) and connect the board with the

Reviewer: ```suggestion
Open a serial terminal (for example Minicom or PuTTY) and connect the board with the
```

following settings:

- Speed: 115200
- Data: 8 bits
- Parity: None
- Stop bits: 1

> Reset the board and the following message will appear on the corresponding
> serial port, one is master another is remote:

Reviewer: ```suggestion
When you reset the development kit, the following messages (one for master and one for remote) will appear on the corresponding serial ports:
```
